readme.md

```diff
@@ -248,8 +246,23 @@ We would also like to distribute additional scenarios with the game, when the ti
 
 ---
 
-# 5. Licence
-**OpenRCT2** is licensed under the GNU General Public License version 3.
+# 5. License
+**OpenRCT2**, an open-source re-implementation of RollerCoaster Tycoon 2.
+
+Copyright (C) 2014 OpenRCT2 contributors
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program. If not, see <http://www.gnu.org/licenses/>.
```

Comment:
There is no need to copy the licence text, but we should point using to the licence file:

```suggestion
**OpenRCT2** is licensed under the GNU General Public License version 3. See the `licence.txt` file for more details.
```